pkg/agent/agentclient/client.go

```diff
@@ -264,8 +264,10 @@ func (a *AgentClient) proxyToRemote(conn net.Conn, connID int64) {
 		for {
 			n, err := conn.Write(d[pos:])
 			if err == nil {
+				klog.Infof("[connID: %d] write last %d data to remote", connID, n)
```

Comment:
Might be worth logging len(d). Default implementation seems like it won't return other than err or n == len(d) but in case...

Reply:
After reading this carefully, I think we shouldn't log len(d). The write is not guaranteed to write the entire d[pos:] to the conn. Writing d in pieces is the purpose of this for loop.